src/deprecated.jl

```diff
@@ -0,0 +1,3 @@
+Base.@deprecate_binding AbstractContext Assertion
+@deprecate Context(; limit=1.0, kwargs...) Threshold(limit)
+@deprecate WeightedContext(wv; limit=1.0, kwargs...) Threshold(limit; weights=wv)
```

Comment:
Does this actually work?
Thats really surprising to me

Reply:
The behaviour isn't identical, but yes? I can add some deprecation tests if you like. Actually, I should also add `false` cause these were never exported.